bin/setup.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+// use process.env to access npm config environment variables.
+// - https://docs.npmjs.com/misc/config#environment-variables
+// - https://docs.npmjs.com/misc/config#save
+var saveRequested = process.env.npm_config_save
+
+var pathToAppRoot = path.resolve('..', '..')
+
+// This block only executes if Hoodie is installed with `-S` or `--save` flags.
+if (saveRequested) {
+  var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+
+  packageJson.scripts = packageJson.scripts || {}
+  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  var newPackageJson = JSON.stringify(packageJson, null, 2)
+
+  fs.writeFile(path.join(pathToAppRoot, 'package.json'), newPackageJson,
+    function (error) {
+      if (error) {
+        console.error(error)
```

Comment:
can you please use `npmlog` like we do in other parts of `hoodie`? And maybe add a bit of an explanation, too?

```js
var log = require('npmlog')
log.error('setup', 'Could not create package.json at ' + path.join(pathToAppRoot, 'package.json')
log.error(error)
```